lib/releaselib.sh

```diff
@@ -580,54 +580,57 @@ release::gcs::locally_stage_release_artifacts() {
   local release_stage=$build_output/release-stage
   local release_tars=$build_output/release-tars
   local gcs_stage=$build_output/gcs-stage/$version
-  local gce_path=$release_stage/full/kubernetes/cluster/gce
   local src
   local dst
-  local gci_path
 
   logecho "Locally stage release artifacts..."
 
   logrun rm -rf $gcs_stage || return 1
   logrun mkdir -p $gcs_stage || return 1
 
-  # GCI path changed in 1.2->1.3 time period
-  if [[ -d $gce_path/gci ]]; then
-    gci_path=$gce_path/gci
-  else
-    gci_path=$gce_path/trusty
-  fi
-
   # Stage everything in release directory
   logecho "- Staging locally to ${gcs_stage##$build_output/}..."
   release::gcs::stage_and_hash $gcs_stage $release_tars/* . || return 1
 
-  # Having the configure-vm.sh script and and trusty code from the GCE cluster
-  # deploy hosted with the release is useful for GKE.
-  release::gcs::stage_and_hash $gcs_stage $gce_path/configure-vm.sh extra/gce \
-   || return 1
-  release::gcs::stage_and_hash $gcs_stage $gci_path/node.yaml extra/gce \
-   || return 1
-  release::gcs::stage_and_hash $gcs_stage $gci_path/master.yaml extra/gce \
-   || return 1
-  release::gcs::stage_and_hash $gcs_stage $gci_path/configure.sh extra/gce \
-   || return 1
+  if [[ "${FLAGS_release_kind}" == "kubernetes" ]]; then
```

Comment:
Nit. The style of this file doesn't use the extraneous {} around variables.  Here and elsewhere.

Reply:
@david-mcmahon, Fixed the nits.